challenge.rb

```diff
@@ -67,6 +102,8 @@ def ac_monitor current_temp, ac_working, desired_temp
 # and print out the result of `fizz_buzz_calculator` for that number.
 # This question will be graded independently of question four.
 
-# Replace this comment with your code.
-
+def fizz_buzz(min,max)
+  puts (min..max).map { |i| fizz_buzz_calculator(i) }
+end
```

Comment:
Not quite the spec - we were supposed to print out each number separately. I probably would have used `each` to accomplish that, with a `puts` inside the block. Still, you're pretty close to the mark.